get_image_predictions.py

```diff
@@ -0,0 +1,19 @@
+import argparse
+import pickle
+from model_prediction import *
+
```

Comment:
here, read this: http://docs.python-guide.org/en/latest/writing/structure/#structure-of-code-is-key and structure your code in a way that makes sense to you after reading that. The exact structure isn't super important - just that there _is_ structure.